apps/prairielearn/src/migrations/20241018171312_questions__share_publicly_backfill.ts

```diff
@@ -1,5 +1,5 @@
 import { enqueueBatchedMigration } from '@prairielearn/migrations';
 
 export default async function () {
-  await enqueueBatchedMigration('20241018171312_questions__share_publicly__backfill.ts');
+  await enqueueBatchedMigration('20241018171312_questions__share_publicly__backfill');
```

Comment:
I assume this is a drive by fix?

Reply:
Yep! It doesn't actually matter as the comment below spells out, but consistency makes copy/pasting boilerplate easier without having to think about why different migrations are different (as I had to do in this PR, lol):

https://github.com/PrairieLearn/PrairieLearn/blob/621af972cc084dd9786e70eb2402d0b605cd7c94/packages/migrations/src/batched-migrations/batched-migrations-runner.ts#L313-L327

Reply:
Thanks for the explanation!